Functions/xASL_bids_BIDSifyFixBasicFields.m

```diff
@@ -0,0 +1,90 @@
+function jsonIn = xASL_bids_BIDSifyFixBasicFields(jsonIn,studyPar,bidsModality)
+%xASL_bids_BIDSifyFixBasicFields Check if required fields exist in studyPar but not in jsonIn 
+% or if we can find them out in other ways
+%
+% FORMAT: jsonIn = xASL_bids_BIDSifyFixBasicFields(jsonIn,studyPar);
+%
+% INPUT:
+%   jsonIn       - JSON with the input fields - from DICOMs (REQUIRED)
+%   studyPar     - Manually defined parameters (REQUIRED)
+%   bidsModality - Modality (CHAR ARRAY, OPTIONAL, DEFAULT = 'asl')
+%
+% OUTPUT: 
+%   jsonIn    - ordered and checked JSON structure
```

Comment:
it's not ordered and checked - It's really only merged

Reply:
Ah sorry, that was copy-pasted from another header c0d6cf4